test.jl

```diff
@@ -1,17 +1,17 @@
 using Test
 using LinearAlgebra
 using Plots
```

Comment:
Not included in the project manifest, make a separate `test` directory with its own environment for this